04-spatial-operations.Rmd

```diff
@@ -803,20 +803,24 @@ Chapter \@ref(gis) shows how to access such GIS functionality from within R.
 
 ### Zonal operations
 
-*Zonal* operations are similar to focal operations.
-The difference is that zonal filters can take on any shape instead of a predefined rectangular window, i.e., the raster cells defining the filter do not have to be necessarily neighors.
+Just like focal operations, *zonal* operations apply an aggregation function to multiple raster cells.
+<!-- Several aspects distinguish zonal from focal operations:
+- A (rectangular) window of neighboring raster cells (e.g., 3x3 cells) usually defines the spatial filter of focal operations.  -->
+However, a second raster, usually a categorical raster, defines the *zonal filters* (or 'zones') in the case of zonal operations as opposed to a predefined neighborhood window in the case of focal operations (see previous Section).
+Consequently, the raster cells defining the zonal filter do not necessarily have to be neighbors.
+<!--Hence zonal filters can take on many more shapes than spatial filters, which always consist of neighbouring cells (in a rectangular or other configuration).-->
 Our grain size raster is a good example (right panel of Figure \@ref(fig:cont-raster)) because the different grain sizes are spread in an irregular fashion throughout the raster.
+Finally, the result of a zonal operation is a summary table grouped by zone which is why this operation is also known as *zonal statistics* in the GIS world. 
```

Comment:
```suggestion
Finally, the result of a zonal operation is a summary table grouped by zone which is why this operation is also known as *zonal statistics* in the GIS world. 
By contrast, focal operations return a raster.
```

Reply:
Looks good to me. Also, first time I've seen 'suggestions' in GitHub. That is awesome!

Reply:
Hi @florisvdh I do not have permission to commit your suggestion. But please implement it and we can accept this PR I think. Heads-up @jannes-m .

Reply:
Done. Seems like a Github bug to me though, as 'edits from maintainers' is marked for my branch.